src/Pillar-PetitPillar.package/PRPillarParser.class/class/parse..st

```diff
@@ -0,0 +1,3 @@
+parsing
+parse: aStream
+	^ self new optimize parse: aStream
```

Comment:
This change is ok

Reply:
thinking about clean code, I prefer to remove it. I put it in case someone will use the parser unoptimized